File: python/rna-transcription/rna_transcription.py
_rna_map = {
  "G": "C",
  "C": "G",
  "T": "A",
  "A": "U"
}

def to_rna(dna_strand):
  transcripted = list(map(lambda nucleotide: _rna_map[nucleotide], dna_strand))

  return ''.join(transcripted)

File: javascript/rna-transcription/rna-transcription.js
const MAPPING = {
  'G': 'C',
  'C': 'G',
  'T': 'A',
  'A': 'U'
};

const conversion = (dna) => MAPPING[dna];

export const toRna = (dnaStrand) => {
  return [...dnaStrand].map(conversion).join('');
};
